components/form/builder/src/index.scss

```diff
@@ -1,10 +1,10 @@
 @import '~@schibstedspain/sui-theme/lib/index';
 @import '~@s-ui/react-atom-spinner/lib/index';
 @import '~@s-ui/react-atom-button/lib/index';
-@import './Checkbox/index';
 @import './Input/index';
 @import './Switch/index';
 @import './Select/index';
+@import './Checkbox/index';
```

Comment:
Select styles are resetting some checkbox styles, so we need checkbox after select import.